sites/www_mozilla_org_frankfurt.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I'd also suffix these files with something like `_native` to make it clearer they don't use throttled connections, unlike the other tests.